### PR TITLE
refactor: make constant protected

### PR DESCRIPTION
**Pull request by gabrielfs7**
refactor: make constant protected

https://oat-sa.atlassian.net/browse/ADF-573

### CODE REVIEW

**Review by shpran — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

**Review by boajer — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

**Review by chinnu-focaloid — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

File: views/package.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
         "@oat-sa/tao-core-libs": "0.4.3",
         "@oat-sa/tao-core-sdk": "1.15.0",
         "@oat-sa/tao-core-shared-libs": "1.0.3",
-        "@oat-sa/tao-core-ui": "1.30.1",
+        "@oat-sa/tao-core-ui": "git+https://github.com/oat-sa/tao-core-ui-fe.git#feat/ADF-595/atomic-roles-assets",
```
Comment (gabrielfs7) [inline views/package.json:21]: - [ ] Do not forget to update before merge

**Comment by 404passionFruit** [comment]
Before merging

- [ ] https://github.com/oat-sa/tao-core-ui-fe/pull/341 should be merged 
- [ ] npm package should be released 
- [ ] the version should be updated in this PR in package.json and package-lock.json

**Pull request merged**